react/src/index.js

```diff
@@ -140,6 +140,15 @@ Sentry.init({
 
     return event;
   },
+  beforeSendTransaction(event) {
+    // ui.action.click transactions are showing up in the demo
+    // which is undesirable
+    if (event?.contexts?.trace?.op === 'ui.action.click') {
```

Comment:
@cstavitsky I believe we since un-did the Experiment for transactions from UI clicks but here's my take on this.

I'm a believer that if no one is specifically using `ui.action.click` transactions, then no, it's not a problem to remove them. When we adopt Beta's and Experiments for the sake of padding our demo data with more data, without a clear defined benefit, then it's opening us up to risk of interference in unforeseen ways.